src/seedu/addressbook/ui/TextUi.java

```diff
@@ -166,7 +169,7 @@ private static String getIndexedListForViewing(List<String> listItems) {
      * @param visibleIndex visible index for this listing
      */
     private static String getIndexedListItem(int visibleIndex, String listItem) {
-        return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
+        return Formatter.formatIndexedListItem(visibleIndex,listItem);
```

Comment:
This belongs to LO5.1. Remember to switch back to master before starting a new feature branch and submit clean PR.